src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  width: calc(#{$cell-size} * 7 + #{$gap-size} * 6 + #{$padding-size} * 2);
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @each $day, $day-index in $weekdays {
+    &--start-day--#{$day} &__cell:first-child {
+      margin-left: (6 - $day-index) * 101px;
```

Comment:
please make monday be the first and not the last
![image](https://github.com/mate-academy/layout_calendar/assets/54936044/cb126791-d1e1-4a23-a6ed-efbf2bc9e8da)